### DISCUSSION TITLE
How to transfer contribution activity from an old account to a new account?

### DISCUSSION THREAD

**Discussion by Tankman61** [community/community#146003] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Already transferred the old email that I used on my old account to commit to my old repos to my new account, however the contributions aren't showing up (which they should according to GitHub's page on this issue). I'd like to transfer my contributions from previous years (2023) on my old account to this current account if possible. Any help would be appreciated!

**Comment by cachedcake** [comment]
I can't say for certainty but I'm pretty sure you're not able to transfer contributions from one account to another.

**Comment by tyagraj777** [comment]
Great question for the day !!!

Transferring contributions from your old GitHub account to a new account can be challenging since GitHub uses email addresses tied to commits to determine contribution ownership. Here’s how you can troubleshoot and ensure contributions appear in your new account:

Steps to Transfer Contributions:
1. Verify Email Address in New Account
Ensure the email address used for the old account’s commits is added and verified in the new account.

-      Go to Settings > Emails in your new account.
-      Add the email address used in your old commits.
-      Verify the email by clicking the confirmation link sent by GitHub.

2. Check Commit Email Address

- Confirm the email address used in your old commits matches the one added to your new account.
- If you used a different email in your .gitconfig or through another method, it must also be added and verified.

To check the email address in the old commits:
$git log --author="Your Name"

3. Ensure Repository Visibility

- Contributions will only appear if:

- The repositories are public.

- Your contributions meet GitHub's contribution guidelines (e.g., creating commits, opening issues, or submitting pull requests).

- If the repositories are private, you need to enable Private Contributions in your new account:
   Go to Settings > Profile > Public Contributions and enable "Include private contributions on my profile."

4. Re-Sync Contribution Data

- GitHub periodically updates contribution graphs. If you’ve recently added the old email to your new account, it may take some time for the contributions to appear.

- To force a refresh, try:

    - Removing and re-adding the email in your new account.
    - Checking after 24-48 hours.

Hope this helps !!!

Thanks

**Reply by mecodeatlas** [reply]
Hey, @Tankman61! Did @tyagraj777's answer help you with what you needed? :)

**Reply by Tankman6** [reply]
Yup, I figured out the problem, its because I kept my email address as private which was why the commits didn't sync

**Reply by Tankman61** [reply]
Is there any way to transfer this private email? 134973493+Tankman6@users.noreply.github.com is the one. I simply turned off keeping my email address private for both accounts.

**Reply by ghost** [reply]
@Tankman6 @Tankman61 I have created a new account and transferred all my forked repos. Also I added the email in my New account that was connected with my old GitHub account. Now what should I do such that I will get my commits in my New account?
